src/UITests/LiveMode.cs

```diff
@@ -37,7 +38,7 @@ private void TestTestResults()
 
         private void ValidateResultsInUIATree()
         {
-            driver.TestMode.ResultsInUIATree.ValidateResults(2, 11);
```

Comment:
>11 [](start = 64, length = 2)

Is there a way to avoid potentially changing these values any time we pick up a new version of Axe.Windows?

Reply:
Per our offline discussion, I've added a comment explaining when we expect this value to change